manifests/blacksmith/external-bosh.yml

```diff
@@ -0,0 +1,30 @@
+---
+meta:
+  default_bosh_exodus_path:     (( concat $GENESIS_ENVIRONMENT "/bosh" ))
+  bosh_exodus_path:             (( grab params.bosh_exodus_path || meta.default_bosh_exodus_path ))
+  bosh_exodus_src:              (( concat $GENESIS_EXODUS_MOUNT meta.bosh_exodus_path ":" ))
+  
+  external_bosh_username: (( vault meta.bosh_exodus_src "blacksmith_user" ))
+  external_bosh_password: (( vault meta.bosh_exodus_src "blacksmith_password" ))
+  external_bosh_address: (( vault meta.bosh_exodus_src "url" ))
+  external_bosh_cacert: (( vault meta.bosh_exodus_src "ca_cert" ))
```

Comment:
Is there any way we can just ask for the bosh deployment and grab these four values from its exodus?  that would be the more genesis-y way of doing things

Reply:
Im following the same logic as the new prometheus-kit as seen here https://github.com/genesis-community/prometheus-genesis-kit/blob/v1.7.0-rc/manifests/monitor-cf-v2.yml